exercises/rail-fence-cipher/src/example.c

```diff
@@ -0,0 +1,54 @@
+#include "rail_fence_cipher.h"
+#include <string.h>
+#include <stdlib.h>
+
+char *encode(char *text, size_t rails)
+{
+   size_t length = strlen(text);
+   char *ciphertext = (char *)calloc(length + 1, sizeof(char));
+   size_t cell_size = 2 * rails - 2;
+   if (!cell_size) {            // kludge
```

Comment:
Don't include kludges. The example should probably be best practice. If checking that `cell_size` is not equal to zero, then do that (it doesn't take many more characters 😉 ), as in 

```
if (cell_size != 0) {        // no kludges here :)
```